changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Payments Changelog ***
 
+= 3.7.0 - 2022-xx-xx =
+*
```

Comment:
I think we need to move the changelog of this PR https://github.com/Automattic/woocommerce-payments/pull/3617 to 3.7.0 as it's not really included in the `release/3.6.0` branch. 

https://github.com/Automattic/woocommerce-payments/commits/release/3.6.0

Reply:
Fixing in a seperate PR